src/QMCDrivers/WFOpt/QMCCostFunctionBatched.cpp

```diff
@@ -281,17 +273,18 @@ void QMCCostFunctionBatched::checkConfigurations()
     {
```

Comment:
Let's not use iw when it's not an index over walkers. I'm probably guilty of it as well but it might as well be i if it doesn't carry any information.

Reply:
One of the next steps will be to operate on batches (rather than the full sample list).  The index variables can be changed then.